src/components/molecules/timezone.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { forwardRef, useCallback, useId, useMemo, useState } from "react";
+
+import { SingleValue } from "react-select";
+import TimezoneSelect from "react-timezone-select";
+
+import { getSelectDarkStyles, getSelectLightStyles } from "@constants";
+import { SelectOption, SelectProps } from "@interfaces/components";
+import { cn } from "@utilities";
+
+import { Hint } from "@components/atoms";
+
+const defaultTimezone = { label: "UTC", value: "UTC" };
+
+interface TimezoneSelectProps extends Omit<SelectProps, "options" | "value" | "onChange"> {
+	value?: string;
+	onChange?: (value: SingleValue<SelectOption>) => void;
+}
+
+export const AkTimezoneSelect = forwardRef<HTMLDivElement, TimezoneSelectProps>(
```

Comment:
The component uses `useMemo` for styles and classes which is good. However, consider memoizing the entire component with `React.memo` if it causes unnecessary re-renders in parent forms.